src/styles/card.scss

```diff
@@ -0,0 +1,106 @@
+$text-color: #616070;
+$button-text-color: #00acdc;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  border: solid 1px #f3f3f3;
+  border-radius: 5px;
+  padding-inline: 16px;
+  transition: all 300ms linear;
+
+  &:hover{
+    transform: scale(1.2);
+  }
```

Comment:
The `:hover` effect applied on the `.card` class makes the card scale up by 1.2 times. This may cause the layout to break or overlap with other elements on the page. It's better to use a subtle effect like box-shadow to indicate hover, and avoid changing layout dimensions.